composer.json

```diff
@@ -23,13 +23,15 @@
     ],
     "require": {
         "php": ">=7.0.0",
-        "symfony/framework-bundle": "^2.0||^3.0||^4.0",
-        "symfony/validator": "^2.0||^3.0||^4.0",
-        "symfony/console": "^2.0||^3.0||^4.0",
-        "dragonmantank/cron-expression": "^2.0"
+        "symfony/framework-bundle": "^5.0",
+        "symfony/validator": "^5.0",
+        "symfony/console": "^5.0",
+        "dragonmantank/cron-expression": "^3.0"
     },
     "require-dev": {
-        "phpunit/phpunit": "5.*"
+        "phpunit/phpunit": "5.*",
+        "symfony/browser-kit": "^5.1",
```

Comment:
why did you add "symfony/browser-kit" and "symfony/css-selector"?  looks like it's not used and can be removed

Reply:
good point, fixed